### PR TITLE
Fake table.c.values() type: List[Column]

### PR DESCRIPTION
**Pull request by sam-alloy**
https://app.clubhouse.io/alloytech/story/40743/typing-for-table-c-values-in-schema-py

This adds `.values() -> List[Column]` to `table.c`

### CODE REVIEW

File: sqlacodegen/codegen.py [hunk 1]
```diff
@@ -647,7 +653,12 @@ class _AccountTable(Table):  # pylint: disable=too-many-ancestors,abstract-metho
         columns_type = f'_{camel_case_name}Columns'
         proxy_type = f'_{camel_case_name}Table'
 
-        rendered = f'class {columns_type}:\n'
+        rendered = dedent(f'''\
+        class {columns_type}:
+            def values(self) -> List[Column]:
+                return []
```
Comment (dsgkirkby) [inline sqlacodegen/codegen.py]: Why can we return an empty list here? Won't this mean that usages of `.values()` will get an empty list, breaking them?

Comment (sam-alloy) [reply sqlacodegen/codegen.py]: We only used the type here, so it should not break anything.
However, I do think having all columns in the list is clearer, on it

Comment (dsgkirkby) [reply sqlacodegen/codegen.py]: Oh wait I see this is just used for the type. Maybe we can make it an `@abstractmethod`?

Comment (sam-alloy) [reply sqlacodegen/codegen.py]: oh that's a good idea, let me test it!

**Review by dsgkirkby — APPROVED** [review]
LGTM!

**Comment by dsgkirkby** [comment]
To avoid a repeat of today, let's merge this to master first and use the sha that's on master in roardata

**Comment by sam-alloy** [comment]
> To avoid a repeat of today, let's merge this to master first and use the sha that's on master in roardata

Yeah... Turns out GitHub does not provide checkout for stale commits, although it was still viewable on the web

**Comment by dsgkirkby** [comment]
Also we shouldn't be using stale commits anyways :)

**Pull request merged**